### PR TITLE
added more to deployment troubleshooting section

### PR DESCRIPTION
**Pull request by murdoaird**
Added new troubleshooting section in response to CentOS 7 provisioning error.

### CODE REVIEW

File: guide/ops/troubleshooting/deployment.md [hunk 1]
```diff
@@ -143,3 +143,25 @@ start tasks are completed).
...
 "Entity's Error Status".
+
+## Invalid packet error
+
+If you receive an error message similar to the one below when provisioning a Centos VM, you can workaround the issue by explicitly setting the user that AMP should use to login to the VM.
+
+`error acquiring SFTPClient() (out of retries - max 50)
+Invalid packet: indicated length too large
+java.lang.IllegalStateException
+Invalid packet: indicated length too large`
```
Comment (drigodwin) [inline guide/ops/troubleshooting/deployment.md]: This should all be {% highlight bash %} rather than this

File: guide/ops/troubleshooting/deployment.md [hunk 2]
```diff
@@ -143,3 +143,25 @@ start tasks are completed).
 
 See the [overview](overview.html) for where to find additional information, especially the section on
 "Entity's Error Status".
+
+## Invalid packet error
+
+If you receive an error message similar to the one below when provisioning a Centos VM, you can workaround the issue by explicitly setting the user that AMP should use to login to the VM.
```
Comment (neykov) [inline guide/ops/troubleshooting/deployment.md]: Ubuntu machines print a similar message on bluebox - better not restrict to centos.

**Comment by drigodwin** [comment]
LGTM, thanks @murdoaird

File: guide/ops/starting-stopping-monitoring.md [hunk 3]
```diff
@@ -3,81 +3,119 @@ title: Starting, Stopping and Monitoring
...
+</ul>
 
-If you are using the `.rpm` or `.deb` package of Apache Brooklyn, then Brooklyn
-will integrate with your OS service management. Commands such as
-`service brooklyn start` will work as expected, and Brooklyn's PID file will be
-stored in the normal location for your OS, such as `/var/run/brooklyn.pid`.
+<div class="tab-content">
+<strong class="hidden started-pdf-include">Vagrant</strong>
 
+Now start the provided brooklyn instance supplying your brooklyn Download credentials in place of the `myuser` and `mypassword` exbrooklynles below
```
Comment (aledsage) [inline guide/ops/starting-stopping-monitoring.md]: Brooklyn doesn't have download credentials.
I presume remove the `user=myuser password=mypassword` below as well.

What is "exbrooklynles"?

File: guide/ops/starting-stopping-monitoring.md [hunk 4]
```diff
@@ -3,81 +3,119 @@ title: Starting, Stopping and Monitoring
...
 
-## Platform-independent distributions
+{% highlight bash %}
+$ user=myuser password=mypassword vagrant up brooklyn
+{% endhighlight %}
 
-The platform-independent distributions are packaged in `.tar.gz` and `.zip`
-files.
+If you do not know your username or password (it should have been provided in your welcome email) please contact <a href="mailto:help@cloudsoft.io">help@cloudsoft.io</a> for assistance.
```
Comment (aledsage) [inline guide/ops/starting-stopping-monitoring.md]: Looks wrong for brooklyn docs.

File: guide/ops/starting-stopping-monitoring.md [hunk 5]
```diff
@@ -3,81 +3,119 @@ title: Starting, Stopping and Monitoring
...
+$ systemctl start brooklyn
+{% endhighlight %}
 
-To stop Brooklyn, simply send a `TERM` signal to the Brooklyn process. The PID
-of the most recently run Brooklyn process can be found in the `pid_java` file at
-the root of the Brooklyn directory.
+The application should then output its logs to `/opt/brooklyn/log/brooklyn.info.log` and `/opt/brooklyn/log/brooklyn.debug.log`.
 
-For example:
+Brooklyn will integrate with your OS service management. Commands such as `service brooklyn start` will work as expected, and Brooklyn's PID file will be stored in the normal location for your OS, such as `/var/run/brooklyn.pid`.
```
Comment (aledsage) [inline guide/ops/starting-stopping-monitoring.md]: I don't think this will be true (`/var/run/brooklyn.pid`) if using brooklyn karaf. We have more work to do for that; currently it would all be written to `/opt/brooklyn/*` I think.

File: guide/ops/starting-stopping-monitoring.md [hunk 6]
```diff
@@ -3,81 +3,119 @@ title: Starting, Stopping and Monitoring
...
+
+Brooklyn will integrate with your OS service management. Commands such as `service brooklyn start` will work as expected, and Brooklyn's PID file will be stored in the normal location for your OS, such as `/var/run/brooklyn.pid`.
+
+</div>
+
+<div id="impl-4" class="tab-pane fade">
+
+<strong class="hidden started-pdf-include">OSX / Linux</strong>
+
+To start Cloudsoft brooklyn (assuming you are in the cloudsoft-brooklyn-karaf-{{ site.brooklyn-version }}/bin directory):
```
Comment (aledsage) [inline guide/ops/starting-stopping-monitoring.md]: Not "Cloudsoft". Search and remove all mention.

File: guide/ops/troubleshooting/deployment.md [hunk 7]
```diff
@@ -143,3 +152,27 @@ start tasks are completed).
...
+If you receive an error message similar to the one below when provisioning a VM, you can workaround the issue by explicitly setting the user that AMP should use to login to the VM (typically the OS default user).
+
+{% highlight bash %}
+error acquiring SFTPClient() (out of retries - max 50)
+Invalid packet: indicated length too large
+java.lang.IllegalStateException
+Invalid packet: indicated length too large
+{% endhighlight %}
+
+An example of how to explicitly set the user is shown below (when defining a Location) by using 'loginUser':
```
Comment (aledsage) [inline guide/ops/troubleshooting/deployment.md]: I'd add something like:

    In most clouds, the initial `loginUser` depends on the VM image. To fix the error above, 
    identify the correct username for the image and supply this explicitly using the config `loginUser`:

File: guide/ops/troubleshooting/deployment.md [hunk 8]
```diff
@@ -143,3 +152,27 @@ start tasks are completed).
 
 See the [overview](overview.html) for where to find additional information, especially the section on
 "Entity's Error Status".
+
+## Invalid packet error
+
+If you receive an error message similar to the one below when provisioning a VM, you can workaround the issue by explicitly setting the user that AMP should use to login to the VM (typically the OS default user).
```
Comment (aledsage) [inline guide/ops/troubleshooting/deployment.md]: I'd reword to something like:

    If you receive an error message similar to the one below when provisioning a VM, it means that the wrong 
    username is being used for ssh'ing to the machine. The "invalid packet" is because a response such as 
    "Please login as the ubuntu user rather than root user." is being sent back.
    
    You can workaround the issue by explicitly setting the user that AMP should use to login to the VM 
    (typically the OS default user).

**Comment by aledsage** [comment]
LGTM; merging.
